src/Distributions/Bernoulli/Correctness.dfy

```diff
@@ -43,19 +43,21 @@ module Bernoulli.Correctness {
     requires n != 0
     requires m <= n
     ensures
-      var e := iset s | Model.Sample(m, n)(s).value;
+      var e := iset s | Model.Sample(m, n)(s).Equals(true);
       && e in Rand.eventSpace
       && Rand.prob(e) == m as real / n as real
   {
     reveal Model.Sample();
-    var e := iset s | Model.Sample(m, n)(s).value;
+    var e := iset s | Model.Sample(m, n)(s).Equals(true);
+    var ltM: nat -> bool := x => x < m;
+    var ltM1: nat -> bool := x => x < m - 1;
```

Comment:
nit: define a higher order function `lt: nat -> nat -> bool` and replace `ltM`, `ltM1` and `x => x < 0` (on line 60).